.travis.yml

```diff
@@ -404,7 +404,7 @@ jobs:
         - bash ./scripts/docker-run-memory-test.sh
 
     # Sanitizer tests
-    - if: (type = cron) OR (branch = prerelease_test)
+    - if: (type = pull_request) OR (branch = prerelease_test)
```

Comment:
I thought we are running this only for cron jobs. Is this temporary?

Reply:
it was only for testing